packages/lodestar/src/chain/factory/duties/index.ts

```diff
@@ -31,8 +31,8 @@ export function generateEmptyAttesterDuty(publicKey: BLSPubkey, duty?: Partial<A
   return {
     validatorPubkey: publicKey,
     aggregatorModulo: 1,
-    attestationSlot: null,
-    committeeIndex: null,
+    attestationSlot: -1,
+    committeeIndex: -1,
```

Comment:
this could be breaking change, not sure how ssz handles negative numbers and our VC don't handle -1 at all, rather put those fields as optional in type